utility/pigpio/interrupt.h

```diff
@@ -1,15 +1,9 @@
-/*
-Interrupts functions extruded from wiringPi library by Oitzu.
-
-wiringPi Copyright (c) 2012 Gordon Henderson
-https://projects.drogon.net/raspberry-pi/wiringpi
-wiringPi is free software: GNU Lesser General Public License
-see <http://www.gnu.org/licenses/>
-*/
-#ifndef __RF24_INTERRUPT_H__
-#define __RF24_INTERRUPT_H__
+/**
+ * Interrupt functions wrapped from pigpio library
+ */
+#ifndef RF24_UTILITY_PIGPIO_INTERRUPT_H_
+#define RF24_UTILITY_PIGPIO_INTERRUPT_H_
 
-#include "RF24_arch_config.h"
 #include <pigpio.h>
 
 #define INT_EDGE_SETUP   0
```

Comment:
`INT_EDGE_SETUP` is specific to wiringPi (used to signify that the IRQ pin is already setup). It has no meaning in Arduino API. The value of `0` is already assigned to `INT_EDGE_RISING`. This should be removed.

Reply:
This is done for all drivers. This wasn't re-defined in wiringPi driver.